javaslang/src/main/java/javaslang/collection/Iterator.java

```diff
@@ -1535,9 +1537,9 @@ default T get() {
 
     @Override
     default Iterator<Seq<T>> grouped(int size) {
```

Comment:
**Note:** I internally based the buffer on our persistent Vector, Scala users a mutable ArrayBuffer. Our version is blazing fast (a little slower than the mutable version). Congrats @paplorinc !!!